src/xenia/cpu/backend/x64/x64_sequences.cc

```diff
@@ -1816,6 +1895,10 @@ struct DIV_I32 : Sequence<DIV_I32, I<OPCODE_DIV, I32Op, I32Op, I32Op>> {
 struct DIV_I64 : Sequence<DIV_I64, I<OPCODE_DIV, I64Op, I64Op, I64Op>> {
   static void Emit(X64Emitter& e, const EmitArgType& i) {
     Xbyak::Label skip;
+    Xbyak::Label neg;
+    Xbyak::Label normalDiv;
+    constexpr const int32_t minusOne{-1};
```

Comment:
const applies implicitly for constexpr.

Reply:
Will get on it, it was an habit in a company I worked for to still specify const after constexpr